components/gitpod-protocol/src/usage.ts

```diff
@@ -36,111 +36,4 @@ export interface BillableSession {
     projectId?: string;
 }
 
-export type BillableWorkspaceType = Omit<WorkspaceType, "probe">;
-
-export const billableSessionDummyData: BillableSession[] = [
```

Comment:
Shall we keep this here an only append to the (for now empty) response from server? It would mean we're showing something in the interim which can be useful for demos but also iterating on the UI.

Reply:
Ah, good idea!